README.md

```diff
@@ -99,8 +99,14 @@ console.log('App bundles', bundles);
 ```js
 const { RealityCaptureClient } = require('forge-server-utils');
 const { FORGE_CLIENT_ID, FORGE_CLIENT_SECRET } = process.env;
-const client = new RealityCaptureClient({ client_id: FORGE_CLIENT_ID, client_secret: FORGE_CLIENT_SECRET });
-const photoscene = await client.createPhotoScene('<scenename>', '<callback>', '<format>', '<scenetype>');
+const recap = new RealityCaptureClient({ client_id: FORGE_CLIENT_ID, client_secret: FORGE_CLIENT_SECRET });
+const options = {
+    scenename: '<scene name>',
+    scenetype: SceneType.Aerial,
+    format: OutputFormat.RecapPhotoMesh,
```

Comment:
I'm guessing the `SceneType` and `OutputFormat` enums should also be included from the _forge-server-utils_ library, right?

Reply:
New commit available with fix